scout/server/blueprints/cases/controllers.py

```diff
@@ -663,6 +665,21 @@ def _append_evaluated_variant_by_type(
     """
     for eval_category, variant_key in CASE_REPORT_VARIANT_TYPES.items():
         if variant_key in var_obj and var_obj[variant_key] is not None:
+            if (
+                eval_category == "classified_detailed"
+            ):  # Append info to display the ACMG VUS Bayesian score / temperature
+                variant_acmg_classifications = store.get_evaluations_case_specific(
+                    document_id=var_obj["_id"]
+                )
+                if variant_acmg_classifications:
+                    var_obj["bayesian_acmg"] = get_acmg_temperature(
+                        set(
+                            [
+                                criterium["term"]
+                                for criterium in variant_acmg_classifications[0].get("criteria")
```

Comment:
Nice! I will note that this seems to fail for some variants/classifications, but I have not figured it out yet. It's not about just having a single-click evaluation, this one had terms.
![Screenshot 2024-12-12 at 14 17 02](https://github.com/user-attachments/assets/54fafae0-da52-4c2e-9854-bd386c11d1f4)

Reply:
Those with manual classification (without criteria, I guess?)

Reply:
Mm, it is an "old" test case, and a busy variant. It doesn't have a current manual, but probably has the events for it from before. Could be some combo of that or deleted classifications? It shows ok on the ACMG classify page so a little annoying. We could test a little on stage perhaps, or on a fresh setup locally?

Reply:
Which case is that. do you still have the link?

Reply:
It’s on my local so I can check things against it. Did you deploy to stage yet, otw I can do that and we check a few!

Reply:
Here is the first variant I tried on stage: 
https://scout-stage.scilifelab.se/evaluations/670ceb0507f2ba4f08471fa6
Does not show temp on 
![Screenshot 2024-12-12 at 15 18 10](https://github.com/user-attachments/assets/7f9db344-dfec-4cbd-aa7c-c040b0c8b7f4)

Reply:
> It’s on my local so I can check things against it. Did you deploy to stage yet, otw I can do that and we check a few!

I did deploy another branch after this one I think

Reply:
Ah never mind, you deployed it already

Reply:
Yes, I've redeployed this one now! Here is another case with the same behaviour:

https://scout-stage.scilifelab.se/cust002/F0007523

Reply:
That variant actually has criteria:

<img width="575" alt="image" src="https://github.com/user-attachments/assets/ab841512-ab17-46d4-9788-8695dd0e3fcb" />

Reply:
The weird thing is that if I classify a local variant the same way, the it works:

<img width="403" alt="image" src="https://github.com/user-attachments/assets/e1eed3d8-1834-46d5-9551-b8b30d789124" />

<img width="435" alt="image" src="https://github.com/user-attachments/assets/4b8104af-3049-4a96-bba2-a8d7bf46828f" />

Reply:
Yeah, as is evident from the evaluation page with lit up criteria! Same with the others. That evaluation is somewhat recent, but I haven't checked if it also has some other old event somewhere.

Reply:
The weird thing is that if I classify a local variant the same way then it works

<img width="484" alt="image" src="https://github.com/user-attachments/assets/1e72f387-6142-4956-b8f3-28b9125ba958" />


<img width="435" alt="image" src="https://github.com/user-attachments/assets/30762cf5-e29f-4183-8cd1-7d98a96e1892" />

Reply:
Concur, and I have not been able to figure out the difference between the ones that fail and the ones that work. I guess both the chain of event-checks to acmg lookup and the `[0]` sorting assumption may be guilty (though the variant above has a date, so something else would have to sneak in). We could add a debug output..

Reply:
I start to fear that it's a variant re-decoration of sort